hs/Database/Database.hs

```diff
@@ -28,7 +29,8 @@ setupDatabase = do setupDistributionTable
                    print "breadth table set up"
                    parseAll
                    seedVideos
-
+                   getDepartments
+                   
```

Comment:
It shows that the change has been made in my branch. I've committed and pushed my changes so could it be that it doesn't recognize it's been updated? Thanks again